client/lib/plugins/store.js

```diff
@@ -248,8 +243,8 @@ PluginsStore = {
 		if ( ! plugins ) {
 			return;
 		}
-		plugins = plugins.filter( _filters.isEqual.bind( this, pluginSlug ) );
-		plugin = plugins.pop();
+
+		plugin = find( plugins, _filters.isEqual.bind( this, pluginSlug ) );
```

Comment:
Yeah the idea is to return only one result (one plugin) since the a few different sites could be using the same plugin and we only wanted to return the info from one of them.